third_party/txt/src/skia/paragraph_skia.h

```diff
@@ -29,7 +29,8 @@ namespace txt {
 class ParagraphSkia : public Paragraph {
  public:
   ParagraphSkia(std::unique_ptr<skia::textlayout::Paragraph> paragraph,
-                std::vector<flutter::DlPaint>&& dl_paints);
+                std::vector<flutter::DlPaint>&& dl_paints,
+                const bool impeller_enabled);
```

Comment:
since this is a bool argument and not a reference or something I don't think making it const changes any semantics?

Reply:
Done.